python-threatexchange/threatexchange/signal_type/signal_base.py

```diff
@@ -64,7 +64,13 @@ def get_content_types(cls) -> t.List[t.Type[content_base.ContentType]]:
     @classmethod
     def get_index_cls(cls) -> t.Type[index.SignalTypeIndex]:
         """Return the index class that handles this signal type"""
-        return TrivialSignalTypeIndex
+        # Confused about which one to start with?
+        # Make a subclass of:
+        #   1. TrivialLinearSearchHashIndex: Just use compare_hash
+        #   2. TrivialLinearSearchMatchIndex: If you are MatchesStr
+        # Or if you only support exact matching (like MD5):
+        #   3. TrivialSignalTypeIndex
```

Comment:
<3

Reply:
Since you already drafted this helpful message, you could be fancy and pass a version of it to the `NotImplementedError` that is thrown